File: app/src/main/java/com/example/miformacionctma/data/Actividad.kt
package com.example.miformacionctma.data

data class Actividad(
    val id: String = "",
    val titulo: String,
    val descripcion: String,
    val aprendiz: String = "APR-01",
    val estado: String = "PENDIENTE",
    val createdAt: String = ""
)

File: pruebas/src/fixtures/actividad.js
export function crearActividadFixture(cambios = {}) {
    return {
        id: 1,
        titulo: "Actividad de prueba",
        descripcion: "Descripción de la actividad de prueba",
        estado: "PENDIENTE",
        prioridad: "MEDIA",
        ...cambios
    };
}